macros/upload_invocations.sql

```diff
@@ -130,8 +130,17 @@
             {% set parsed_inv_args_vars = fromyaml(invocation_args_dict.vars) %}
             {% do invocation_args_dict.update({'vars': parsed_inv_args_vars}) %}
         {% endif %}
+
+        {# Only pull the required invocation args keys to avoid non-text keys #}
+        {% set keys_to_find =  ["event_buffer_size", "indirect_selection", "no_print", "partial_parse", "printer_width", "profiles_dir", "quiet", "rpc_method", "select", "send_anonymous_usage_stats", "static_parser", "use_colors", "version_check", "which", "profile", "defer", "exclude", "full_refresh", "write_json", "resource_types", "state", "target", "cache_selected_only", "compile"] %}
```

Comment:
Based on my testing for https://github.com/brooklyn-data/dbt_artifacts/pull/327, this section should run before the [invocation_args_dict.vars block](https://github.com/brooklyn-data/dbt_artifacts/pull/326/files#diff-a3cf74c0edc5850d373b360c2210ab3c88a64d92931abce74dbb377d11a00403L128-R132). If this is not done, passing in vars can cause the error "a string or stream input is required". If needed, I've got time tomorrow to find an illustrative example and create an issue.

Reply:
Hi @d-cole appreciate you looking into this! [We merged a PR](https://github.com/brooklyn-data/dbt_artifacts/pull/338/files) that will handle this section a slightly different way so this logic is out of date.